libs/category/tsconfig.json

```diff
@@ -9,6 +9,24 @@
       "@daffodil/category": [
         "libs/category/src"
       ],
+      "@daffodil/category/state": [
```

Comment:
needs one for `@daffodil/category/routing`

Reply:
good catch, changed